abb_irb6600_support/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(abb_irb6600_support)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+if (CATKIN_ENABLE_TESTING)
+  find_package(roslaunch REQUIRED)
+  roslaunch_add_file_check(tests/roslaunch_test.xml)
+endif()
+
+foreach(dir config launch meshes urdf)
+   install(DIRECTORY ${dir}/
+      DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
+endforeach()
```

Comment:
Also: this can be simplified:
```make
install(DIRECTORY config launch meshes urdf
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

No need for the for-loop anymore.